lib/stats/stats-counter.h

```diff
@@ -72,6 +73,15 @@ stats_counter_get(StatsCounterItem *counter)
   return result;
 }
 
+static inline gchar *
+stats_counter_get_name(StatsCounterItem *counter)
+{
+  if (counter && counter->name)
+    return counter->name;
```

Comment:
It also has a `stats_counter_get` which returns the value. Another getter for the name was created to have a consistent interface for `StatsCounterItem`.